examples/basic-webpack/index.jsx

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var ExampleApplication = React.createClass({
+  render: function() {
+    var elapsed = Math.round(this.props.elapsed  / 100);
+    var seconds = elapsed / 10 + (elapsed % 10 ? '' : '.0' );
+    var message = 'React has been successfully running for ' + seconds + ' seconds.';
```

Comment:
Any reason u prefer vars instead of cons? And what do you think about combining multiple vars together to have single var statement?

thank you!